reflex/app.py

```diff
@@ -853,6 +860,7 @@ def get_compilation_time() -> str:
         if not self._should_compile():
             return
 
+        self._init_var_dependency_dicts()
```

Comment:
Note to myself: I haven't tested it yet, but this may cause issues with backend only workers. May need to call `_init_var_dependency_dicts` in backend only mode after importing the app module

Reply:
I dropped https://github.com/reflex-dev/reflex/commit/f3a98f35425a61d4c2384da715b1a3eb21b8622b again. We would need some kind of hot reload hook to get it working - it's probably not worth it